src/app/carbon-estimator-form/carbon-estimator-form.component.html

```diff
@@ -14,9 +18,9 @@
           required
           [attr.aria-describedby]="(headCount | invalidated) ? 'headCountError' : null" />
         @if (headCount | invalidated) {
-          <div class="tce-error-box tce-flex tce-gap-1" aria-live="polite" id="headCountError">
+          <div class="tce-error-box tce-flex tce-items-center tce-gap-1" aria-live="polite" id="headCountError">
```

Comment:
Should this class be added to the other errors?

Reply:
Yes that seems like a good idea. I didn't add it initially because it was causing issues when some of the flex boxes contained anchor tags but they've been removed now so I've added that class to all the error boxes.